docs/en/connector-v2/sink/common-options.md

```diff
@@ -2,24 +2,15 @@
 
 > Common parameters of sink connectors
 
-|       name        |  type  | required | default value |
-|-------------------|--------|----------|---------------|
-| source_table_name | string | no       | -             |
-| parallelism       | int    | no       | -             |
+|       Name        |  Type  | Required | Default |                                                                                                                                     Description                                                                                                                                      |
+|-------------------|--------|----------|---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| source_table_name | String | No       | -       | When `source_table_name` is not specified, the current plug-in processes the data set `dataset` output by the previous plugin in the configuration file <br/> When `source_table_name` is specified, the current plug-in is processing the data set corresponding to this parameter. |
```

Comment:
Some important caveat should be added: `source_table_name` and `result_table_name` are only used to represent the upstream and downstream relationships of the connector. They should not be treated as in-memory datasets. No data is stored in them.